my_game.py

```diff
@@ -58,22 +57,51 @@ def __init__(self, center_x, center_y, lives, scale):
         # Graphics to use for Player
         super().__init__("images/playerShip1_red.png")
 
-        self.speed = 1
         self.angle = 0
         self.lives = lives
         self.scale = scale
-        self.change_x = self.speed * math.cos(self.angle)
-        self.change_y = self.speed * math.cos(self.angle)
         self.center_x = center_x
         self.center_y = center_y
 
+    def thrust(self):
+        """
+        increase speed in the direction pointing
+        """
+
+        dir_in_quarter = self.angle % 90 / 100
+
+        to_add = PLAYER_SPEED * dir_in_quarter
+
+        if self.angle >= 270:
+            self.change_y += to_add
+            self.change_x += PLAYER_SPEED - to_add
+
+        elif self.angle >= 180:
+            self.change_x += to_add
+            self.change_y -= PLAYER_SPEED - to_add
+
+        elif self.angle >= 90:
+            self.change_y -= to_add
+            self.change_x -= PLAYER_SPEED - to_add
+
+        else:
+            self.change_x -= to_add
+            self.change_y += PLAYER_SPEED - to_add
+
```

Comment:
You need to use _math.cos()_ and _math.sin()_ to calculate _change_x_ and _change_y_. Those functions take radians, not degrees (like self.angle), so use _self.radians_ (Same as self.angle, but in radians).

https://api.arcade.academy/en/latest/api/sprites.html#arcade.Sprite.radians